Fixating position of preview tab's reset button to bottom right corner.

hoangviet1993 opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
<!--
  - Explain what your PR does. If this PR fixes an existing bug, please include 
  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue 
  - when this PR is merged.
  -->
This PR fixes #5092 which was introduced by my 3301699 commit. Some before and after pics:
Before (current develop) : 
![screen shot 2018-07-12 at 11 35 58 am](https://user-images.githubusercontent.com/11153258/42643834-1b34b9dc-85c8-11e8-8980-9aa56e81ce0a.png)
After fix : 
![screen shot 2018-07-12 at 11 33 29 am](https://user-images.githubusercontent.com/11153258/42643844-2154697a-85c8-11e8-91a9-a53a61ee5e40.png)


## Checklist
- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes.
- [ ] The PR explanation includes the words "Fixes #bugnum: ...".
- [ ] The linter/Karma presubmit checks have passed.
  - These should run automatically, but if not, you can manually trigger them locally using `python scripts/pre_commit_linter.py` and `bash scripts/run_frontend_tests.sh`.
- [ ] The PR is made from a branch that's **not** called "develop".
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- [ ] The PR is assigned to an appropriate reviewer.
  - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer.
  - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue.

hoangviet1993 committed:
Changing position property of bottom right container to fixed

DubeySandeep commented on core/templates/dev/head/pages/exploration_editor/preview_tab/preview_tab.html:
```diff
@@ -32,7 +32,7 @@
     bottom: 5px;
     display: block;
-    position: absolute;
+    position: fixed;
```
Why fixed positioning?
I mean is it working when the learner page have scrollable length? 
(I'll do the manual test once, Idk why my develop branch, is falling downloading some third_party dependencies)
Also, check when we have supplement cards! (Like image interaction)

FYI, currently, I can see in prod site refresh button use to be at the end of the preview tab (take that as reference).

DubeySandeep commented on core/templates/dev/head/pages/exploration_editor/preview_tab/preview_tab.html:
```diff
@@ -20,7 +20,7 @@
       </table>
     </div>
-    <button class="btn-lg" ng-click="resetPreview()">
+    <button class="btn-lg " ng-click="resetPreview()">
```
is that an expected change?

hoangviet1993 replied:
Fixed! :+1:

DubeySandeep commented:
Follow up: (on Image interaction)
![image](https://user-images.githubusercontent.com/16653571/42664936-0d6a0314-865b-11e8-80e3-f5c227e19c1a.png)

hoangviet1993 committed:
Adding z-index, explanation and adjusting bottom px

hoangviet1993 commented:
I have to use `position: fixed` because of the added `<div ng-if='Iframed'>`.  A small interaction will have a small preview tab and thus the button floats up.
I have added some explanation and a z-index so it stays on top of other div. PTAL @DubeySandeep

hoangviet1993 commented:
![screen shot 2018-07-12 at 10 23 53 pm](https://user-images.githubusercontent.com/11153258/42669226-533b54b2-8622-11e8-96b3-e9eb420ca884.png)

hoangviet1993 commented:
Note, the button will remain visible at all time. Are we concerned about people previewing exploration on small phone?

seanlip commented:
No we aren't. The exploration editor page is not designed for mobile, so I think we're OK there.

@DubeySandeep feel free to merge if you're happy with this, and I can cherrypick it.

Thanks @hoangviet1993!

DubeySandeep reviewed (approved):
@hoangviet1993, Although these changes LGTM! but it's not like the earlier behaviour where we use to have the `restart from the beginning` button at the end of the page! (see preview tab of this [exploration](https://www.oppia.org/create/MjZzEVOG47_1?collection_id=4UgTQUc1tala#/preview/Introduction))

Also, I didn't find any issue with this design, so I'm going to merge this PR!

hoangviet1993 commented:
I can adjust the button location should any user wanting the original design back. 

Should that happen, it would definitely require me moving the "Restart from beginning" button to base.html, which I do not think makes sense systematically. It would also need more code written to support the button from base.js. 

Overall, although I could not achieve restoring the original look, this PR is still a good compromise imo.
